egs2/jsut/tts1/conf/tuning/train_tacotron2.yaml

```diff
@@ -52,21 +53,25 @@ optim_conf:           # keyword arguments for selected optimizer
 ##########################################################
 #                OTHER TRAINING SETTING                  #
 ##########################################################
-max_epoch: 10               # number of epochs
-grad_clip: 5.0              # gradient clipping norm
+max_epoch: 200              # number of epochs
+grad_clip: 1.0              # gradient clipping norm
 grad_noise: false           # whether to use gradient noise injection
 accum_grad: 1               # gradient accumulation
-batch_bins: 500000          # batch size
+batch_bins: 1000000         # batch bins (for feats_type=fbank)
+# batch_bins: 3750000       # batch bins (for feats_type=raw, *= n_shift / n_mels)
 batch_type: numel           # how to make batch
 sort_in_batch: descending   # how to sort data in making batch
 sort_batch: descending      # how to sort created batches
 num_workers: 1              # number of workers of data loader
 train_dtype: float32        # dtype in training
 log_interval: null          # log interval in iterations
-keep_nbest_models: 1        # number of models to keep
+keep_nbest_models: 5        # number of models to keep
```

Comment:
Do you intend to use averaged model?
Which is better averaged model and best model? Just a question.

Reply:
Yes. Actually, I have not yet compared in detail, but I want to check it.